_includes/v21.2/cdc/avro-limitations.md

```diff
@@ -1,3 +1,7 @@
+- [Decimals](decimal.html) must have precision specified.
+- [`BYTES`](bytes.html) (or its aliases `BYTEA` and `BLOB`) is often used to store machine-readable data. When you stream these types through a changefeed with [`format=avro`](create-changefeed.html#format), CockroachDB does not encode or change the data. However, Avro clients can often include escape sequences so the data is in a printable format, which can interfere with deserialization. A potential solution is to hex-encode `BYTEA` values when initially inserting them into CockroachDB. This will ensure that you can consistently decode the hexadecimal. Note that this process will increase record size.
```

Comment:
> ... is often used to store machine-readable data. When you stream these types ...

nit: `is often` and `these types` should match as either singular or plural.

> ... escape sequences so the data is  ...

nit: `so the data is` --> `to present the data`

> ... hex-encode `BYTEA` values ...

`BYTEA` --> `BYTES` or `BYTES (or its aliases BYTEA and BLOB)` -- to match the main variant we present initially in this UL entry.

> This will ensure that you can consistently decode the hexadecimal.

nit: `you` --> `Avro clients`

> Note that this process will increase record size


optional nit: "this process" could potentially mean using `BYTES` in general. Tiny clarification could sort: `this process` --> `hex-encoding values at insertion`, or something similar. Very nit, very optional.